pkg/xray/provider.go

```diff
@@ -20,33 +24,36 @@ func Provider() *schema.Provider {
 			"url": {
 				Type:     schema.TypeString,
 				Optional: true,
-				DefaultFunc: schema.MultiEnvDefaultFunc([]string{"JFROG_URL", "ARTIFACTORY_URL", "PROJECTS_URL"},
+				DefaultFunc: schema.MultiEnvDefaultFunc([]string{"JFROG_URL", "ARTIFACTORY_URL"},
```

Comment:
No `ARTIFACTORY_URL` I think I said this in the last PR

Reply:
ok